internal/cephes/cephes.go

```diff
@@ -15,11 +17,14 @@ https://lists.debian.org/debian-legal/2004/12/msg00295.html
 */
 
 var (
-	badParamOutOfBounds = "cephes: parameter out of bounds"
+	badParamOutOfBounds         = "cephes: parameter out of bounds"
+	badParamUnderflow           = "cephes: underflow error"
```

Comment:
`igamFac` in `internal/cephes/igam.go` still uses it. Should I have that return 0 in that situation?